Misc/NEWS.d/next/Security/2023-03-06-22-48-08.gh-issue-102153.eiaVrE.rst

```diff
@@ -0,0 +1 @@
+fix CVE-2023-24329
```

Comment:
Fix the parsing problem(CVE-2023-24329) in urlparse when the entire URL
starts with blank characters. This vulnerability would help an attacker to bypass
the protections set by the developer for scheme and host.

Reply:
It's very great.